homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -48,111 +47,93 @@ class AmcrestSensorEntityDescription(BinarySensorEntityDescription):
 SCAN_INTERVAL = timedelta(seconds=BINARY_SENSOR_SCAN_INTERVAL_SECS)
 _ONLINE_SCAN_INTERVAL = timedelta(seconds=60 - BINARY_SENSOR_SCAN_INTERVAL_SECS)
 
-_AUDIO_DETECTED_KEY = "audio_detected"
-_AUDIO_DETECTED_POLLED_KEY = "audio_detected_polled"
-_AUDIO_DETECTED_NAME = "Audio Detected"
-_AUDIO_DETECTED_EVENT_CODES = {"AudioMutation", "AudioIntensity"}
+AUDIO_DETECTED_KEY = "audio_detected"
+AUDIO_DETECTED_POLLED_KEY = "audio_detected_polled"
+AUDIO_DETECTED_NAME = "Audio Detected"
+AUDIO_DETECTED_EVENT_CODES = {"AudioMutation", "AudioIntensity"}
```

Comment:
Do you need all these constants, they seem only used on one place? While using the `EntityDescription`, I would just add them inline in the entity description. Or at least combine the four constants to one typed dataclass / tuple.

Reply:
I missed indeed that they were used in the Config Flow as well. Not necessary to change it now, but in a follow-up PR, I think a dataclass or tuple would be better. Eventually this could even be moved to the pypi library.